### DISCUSSION TITLE
502 Bad gateway after do-release-upgrade Ubuntu

### DISCUSSION THREAD

**Discussion by ragman1976** [netbox-community/netbox#9863] [Help Wanted!] [answered]
Hi,

I upgraded my Ubuntu (Proxmox LXC) from Ubuntu 21.10 'impish' to Ubuntu 22.04 'Jammy Jellyfish'

After that upgrade I got "503 Bad Gateway" error.

How can I fix that?

Greets

**Comment by candlerb** [comment, marked as answer]
You have a problem, but the only symptom you've shown ("Bad Gateway" error) simply means that Netbox is not running.

To debug your problem, I'd suggest you start by checking `systemctl status netbox` and `journalctl -eu netbox`.

*If* the problem is that the version of python3 supplied has changed between Ubuntu 21.10 and 22.04, then the solution may be as simple as re-running `/opt/netbox/upgrade.sh` which will rebuild your python virtualenv.

**Reply by ragman1976** [reply]
Hi,

/opt/netbox/upgrade.sh did the trick :-)

Thanks

**Reply by tumtumsback** [reply]
omg @candlerb ily <3
